regulations/static/regulations/js/source/views/header/preamble-head-view.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * preamble-head-view.js
+ *
+ * This view specifically handles Preamble actions in the
+ * Sub View Header, notably click actions of the read/write tabs.
+ *
+ */
+
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var CommentEvents = require('../../events/comment-events');
+
+var PreambleHeadView = Backbone.View.extend({
+  el: '.preamble-header',
+
+  events: {
+    'click .read-proposal': 'readProposal',
+    'click .write-comment': 'writeComment'
+  },
+
+  initialize: function() {
+    this.$readTab = this.$el.find('.read-proposal');
+    this.$writeTab = this.$el.find('.write-comment');
+
+    this.listenTo(CommentEvents, 'comment:readTab', this.readTabOpen);
+    this.listenTo(CommentEvents, 'comment:writeTab', this.writeTabOpen);
+  },
+
+  readTabOpen: function () {
+    this.$readTab.addClass('active-mode');
+    this.$writeTab.removeClass('active-mode');
+  },
+
+  readProposal: function() {
+    this.readTabOpen();
+
+    CommentEvents.trigger('read:proposal');
+  },
+
+  writeTabOpen: function() {
+    this.$writeTab.addClass('active-mode');
+    this.$readTab.removeClass('active-mode');
+  },
+
+  writeComment: function() {
+    this.writeTabOpen();
+
+    CommentEvents.trigger('comment:write');
```

Comment:
We can defer this to a later patch, but I think we can come up with clearer event names--I don't think I set a good standard so far! For example, it's hard to understand the difference between "comment:write" and "comment:writeTab" without reading the source.